### ISSUE TITLE
Rightclick movers are not on top

### ISSUE THREAD

**Issue by Jeffroiscool** [ForgeUI/ForgeUI#251]
When rightclicking on some stuff it can go under the other borders, see below as example.

![image](https://cloud.githubusercontent.com/assets/4972345/16815963/658f8fea-493c-11e6-8d9b-16a5a747eba4.png)

**Comment by adamjedlicka** [comment]
Yeah I know about this. Unfortunatelly i can't do much about this. I have an idea which I'll try tommorow but I don't promise anything.

**Comment by veex-ua** [comment]
BTW the solution I found for nameplates might apply here. Basically movers parent window must have "FixedHudStratumLow" or normal/high not sure, and then movers have wnd:ToFront() method in their OnShow() event.

**Comment by adamjedlicka** [comment]
Problem with movers is that some have to be over others (players resource bar is on top of unit frame and is higher on z-axis). Atm every mover is in the same stratum as its parent UI element to make sure its all OK and I guess it could be done that every mover is in the same stratum somhow. I was more thinking about separating right-click "precision" menu from mover and when right-clicking mover that menu would get binded to that mover and becase its not directly movers child it can be in different stratum.
